hgweb/command-runner.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Define the list of allowed commands
+allowed_commands=("verify")
+
+# Get the project code and command name from the URL
+IFS='/' read -ra PATH_INFO <<< "$PATH_INFO"
```

Comment:
I think it would be better to have `read` put its result into a variable named something other than `PATH_INFO`, mabe something like `PATH_SEGMENTS`, so that we don't overwrite the `PATH_INFO` variable. We might want to log the value of `PATH_INFO` later on in an error handler.